code/modules/virus2/centrifuge.dm

```diff
@@ -29,8 +29,8 @@
 
 /obj/machinery/computer/centrifuge/update_icon()
 	..()
-	if(! (stat & (BROKEN|NOPOWER)) && (isolating || curing))
-		icon_state = "centrifuge_moving"
+	if(! (stat & (BROKEN|NOPOWER)))
+		icon_state = (isolating|curing ? "centrifuge_moving" : "centrifuge")
```

Comment:
`isolating|curing` needs to be `isolating || curing`.

`|` on its own is a mathematical operator. Specifically a discrete OR statement. `||` is a logical operator, a logical OR.

The former will produce a number. The latter will produce TRUE or FALSE.

Reply:
Technically `||` does not return TRUE or FALSE.